src/components/layout/header/Column.jsx

```diff
@@ -44,11 +44,17 @@ export const Column = ({
 
     const visibleColumns = columns.filter((_col) => !_col.hidden);
 
+    const sortedColumn = columns.filter((_col) => _col.sortDirection);
+
+    const shouldShowCaret = sortedColumn.length > 0
```

Comment:
Since we're using `find` instead of `filter`, we no longer need to check on length, just that the value exists.